tools/config-generator/types.go

```diff
@@ -53,3 +72,21 @@ func NewTestGridMetaData() TestGridMetaData {
 		nonAligned: make([]NonAlignedTestGroup, 0),
 	}
 }
+
+func NewGoVersion(version string) GoVersion {
```

Comment:
Do we need this function? Seems it's unused.

Reply:
It is currently unused, but we should transition the config input template from hard-coded "Go114" and "Go113" keys to `GoVersion: "go1.14"` or similar, and wanted to have a validator

Reply:
Got it.